pandas/io/formats/templates/html.tpl

```diff
@@ -14,7 +14,7 @@
 {% block before_cellstyle %}{% endblock before_cellstyle %}
 {% block cellstyle %}
 {%- for s in cellstyle %}
-    #T_{{uuid}}{{s.selector}} {
+    {%- for selector in s.selectors -%}{%- if not loop.first -%},{%- endif -%}#T_{{uuid}}{{selector}}{%- endfor -%} {
```

Comment:
It's been a while since I've looked at this, but I'm a bit confused. What is this section doing? Is it writing the css styles? Does it matter what order the selector vs the id `#T_uuid` goes in?

Reply:
This is just a comma-separated list of id selectors. Comma is "or", the order doesn't matter.